checkout/Model/DefaultConfigProvider.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * Copyright © Pronko Consulting (https://www.pronkoconsulting.com)
+ * See LICENSE for the license details.
+ */
+declare(strict_types=1);
+
+namespace Pronko\LiqPayCheckout\Model;
+
+use Magento\Checkout\Model\ConfigProviderInterface;
+use Magento\Payment\Model\Method\Adapter;
+use Pronko\LiqPayApi\Api\Data\PaymentMethodCodeInterface;
+
+/**
+ * Class DefaultConfigProvider
+ */
+class DefaultConfigProvider implements ConfigProviderInterface
+{
+    /**
+     * @var Adapter
+     */
+    private $paymentMethodAdapter;
+
+    /**
+     * DefaultConfigProvider constructor.
+     * @param Adapter $paymentAdapter
+     */
+    public function __construct(
+        Adapter $paymentAdapter
+    ) {
```

Comment:
Please inject ConfigInterface from the Payment module instead of the Adapter class

Reply:
@mcspronko Redid it.